cdap-app-fabric/src/main/java/io/cdap/cdap/master/environment/k8s/MasterEnvironmentMain.java

```diff
@@ -102,15 +104,23 @@ public static void doMain(String[] args) throws Exception {
       }
 
       CConfiguration cConf = CConfiguration.create();
+      File cConfFile = new File(CONFIGMAP_MOUNTPATH, "cConf.xml");
+      if (cConfFile.exists()) {
+        cConf.addResource(cConfFile.toURI().toURL());
+      }
+
       SConfiguration sConf = SConfiguration.create();
       if (options.getExtraConfPath() != null) {
-        cConf.addResource(new File(options.getExtraConfPath(), "cdap-site.xml").toURI().toURL());
         sConf.addResource(new File(options.getExtraConfPath(), "cdap-security.xml").toURI().toURL());
       }
 
       SecurityUtil.loginForMasterService(cConf);
 
       Configuration hConf = new Configuration();
+      File hConfFile = new File(CONFIGMAP_MOUNTPATH, "hConf.xml");
```

Comment:
I thought there were a bunch of other config files that also need to be localized (sConf.xml, logback.xml, etc). Where are they used? No changes are required for them?